sql/core/src/main/scala/org/apache/spark/sql/execution/joins/SortMergeJoinExec.scala

```diff
@@ -615,6 +620,7 @@ case class SortMergeJoinExec(
     }
 
     s"""
+       |$joinedRow = new JoinedRow();
```

Comment:
Since we generate Java program, is it necessary to declare `JoinedRow` type for `$joinedRow`?

Reply:
Yea, I think we need.

Reply:
Yeah, that's causing the test failures. This is a typo from some restructuring I did to get this upstream.

Reply:
Fixed.